src/applications/hca/actions.js

```diff
@@ -6,17 +6,31 @@ import { HCA_ENROLLMENT_STATUSES } from './constants';
 // flip the `false` to `true` to fake the endpoint when testing locally
 const simulateServerLocally = environment.isLocalhost() && false;
 
+// action types related to calling /health_care_applications/enrollment_status
 export const FETCH_ENROLLMENT_STATUS_STARTED =
   'FETCH_ENROLLMENT_STATUS_STARTED';
 export const FETCH_ENROLLMENT_STATUS_SUCCEEDED =
   'FETCH_ENROLLMENT_STATUS_SUCCEEDED';
 export const FETCH_ENROLLMENT_STATUS_FAILED = 'FETCH_ENROLLMENT_STATUS_FAILED';
+
+// action types related to calling GET /notifications/dismissed_statuses
+export const FETCH_DISMISSED_HCA_NOTIFICATION_STARTED =
```

Comment:
I keep trying to come with alternative names for these since they feel slightly off to me, but I think it's really that this api endpoint is just weird. I might do `NOTIFICATION_STATUS` instead of just `NOTIFICATION`, since you're not fetching an actual notification, but I think it reads ok either way.